zetaclient/context/app.go

```diff
@@ -82,6 +91,18 @@ func (a *AppContext) GetBTCChainAndConfig() (chains.Chain, config.BTCConfig, boo
 	return btcChain, btcConfig, true
 }
 
+// GetSolanaChainAndConfig returns solana chain and config if enabled
+func (a *AppContext) GetSolanaChainAndConfig() (chains.Chain, config.SolanaConfig, bool) {
```

Comment:
Can we unify `GetEVMChain()`, `GetBTCChainAndConfig()`, and this method into a generic structure? Otherwise we have a lot of hard-coded stuff per specific chain implementation

Reply:
The previous dynamic chain activation PR refactored the config and removed this method and a few others. Since that PR is not merged yet. We can just skip and leave the refactor when it merges.